package.json

```diff
@@ -19,7 +19,8 @@
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each",
-    "test:node": "mocha node-tests/**/*-test.js"
+    "test:node": "mocha node-tests/**/*-test.js",
+    "format": "prettier --write \"./**/*.{md,json,js,yml}\""
```

Comment:
I would prefer to setup Prettier as ESLint plugin as described in the RFC: https://emberjs.github.io/rfcs/0628-prettier.html#detailed-design My main motivation is that we don't need to change much when this RFC is implemented in Ember CLI.